### DISCUSSION TITLE
JSON RPC use

### DISCUSSION THREAD

**Discussion by joydashy** [jesec/rtorrent#48] [Q&A]
Hi there,

Have been trying to work out how to switch to JSON-RPC. I am using Flood 4.7.0 and rTorrent v0.9.8-r16. There's some hinting at JSON RPC in some places, but I can't find any documentation or such on how to start using it.

Thanks!

**Comment by logical-and** [comment]
Same question

**Comment by kannibalox** [comment]
Is there anything indicating to you that JSON-RPC is not in effect? There's nothing that needs to be done on the rTorrent end to enable things, it supports serving both JSON-RPC and and XMLRPC over the same SCGI socket. It looks like flood [tries JSON-RPC first and downgrades only if it's not available](https://github.com/jesec/flood/commit/38605915c65e728bfb6c71c24b56d8804f297370), so there shouldn't be anything to do there either.

**Reply by joydashy** [reply]
Frankly this question is now so old I don't remember why I asked it...
